compiler/dyno/lib/resolution/Resolver.cpp

```diff
@@ -1774,6 +1774,79 @@ void Resolver::exit(const TupleDecl* decl) {
   exitScope(decl);
 }
 
+bool Resolver::enter(const Range* range) {
+  return true;
+}
+void Resolver::exit(const Range* range) {
+  const RecordType* rangeType = RecordType::getRangeType(context);
```

Comment:
This is good for now but in the future I think we will need it to call `chpl_build_bounded_range`. Could you please add a comment to that effect?